kedro-datasets/kedro_datasets/spark/spark_dataset.py

```diff
@@ -114,6 +118,20 @@ def _dbfs_exists(pattern: str, dbutils: Any) -> bool:
         return False
 
 
+def _deployed_on_databricks() -> bool:
+    """Check if running on Databricks."""
```

Comment:
https://github.com/kedro-org/kedro/blob/1c6d108b56e405364be3cf1d9cfca65fe6606096/kedro/framework/project/__init__.py#LL217-L218C87

This should be a function now to determine whether the project is run in `Databricks`.

Reply:
Do you mean we should add a function to Kedro in a separate PR and import that here?

Reply:
Yes since they are doing the exact same thing, we should just keep the logic consistent. If this is a more pressing bug fix I think it's fine to keep this in the PR but we should still refactor it in `kedro` later.